lib/common/config/config.ts

```diff
@@ -11,6 +11,16 @@ export const getConfig = () => {
   return config;
 };
 
+export const setConfigKey = (key: string, value: unknown) => {
+  config[key] = value;
+};
+
+export const setConfigAndReturnOriginalConfigForTestOnly = (configObject) => {
+  const originalConfig = Object.assign({}, config);
+  config = configObject;
+  return originalConfig;
+};
```

Comment:
🤔  we might move this into test `*.test.ts` files, because it's used only in tests.

Reply:
done